include/mbedtls/ssl.h

```diff
@@ -3860,6 +3860,26 @@ void mbedtls_ssl_conf_sni( mbedtls_ssl_config *conf,
 int mbedtls_ssl_set_hs_ecjpake_password( mbedtls_ssl_context *ssl,
                                          const unsigned char *pw,
                                          size_t pw_len );
+
+/**
+ * \brief          Set the EC J-PAKE opaque password for current handshake.
+ *
+ * \note           An internal copy is made, and destroyed as soon as the
```

Comment:
Ok, see the discussion I started on Slack: I'm not sure we need to make a defensive copy here, but let's see what others think.

Reply:
Oh, reading the code I see what I said on Slack was wrong: the key is used right away, in that we pass it to `psa_pake_set_password_key()`. That function currently makes a copy, but I don't think it's guaranteed that it will in the future - it could also just lock the key.

Reply:
If I understood correctly, according to the Slack discussion, I can remove the copy. Am I correct?

Reply:
Yes. Actually, if I read the code correct, it shouldn't change anything in the code, just let's not mention it in the documentation. (Actually we should be explicit and mention that the caller must not destroy the key before the handshake is over.)